src/main/java/io/kestra/plugin/googleworkspace/drive/Download.java

```diff
@@ -36,6 +37,14 @@
                     fileId: "1Dkd3W0OQo-wxz1rrORLP7YGSj6EBLEg74fiTdbJUIQE"
                 """
         )
+    },
+    metrics = {
+        @Metric(
+            name = "size",
+            type = COUNTER.type,
```

Comment:
This doesn't compile, did you check the compilation 😬 ?

```suggestion
            type = Counter.TYPE,
```